core/scss/components/brand-bar/_brand-bar.scss

```diff
@@ -0,0 +1,32 @@
+@charset "UTF-8";
+
+//
+// Brand Bar
+//
+// Stanford brand bar with wordmark logo. This sits at the very top of a website.
+//
+// .su-brand-bar--default - Default brand bar with white wordmark on cardinal red background
+// .su-brand-bar--bright - White wordmark on bright red background
+// .su-brand-bar--dark - White wordmark on dark grey background
+// .su-brand-bar--white - cardinal red wordmark on white background
+//
+// Markup: ../../templates/components/brand-bar/brand-bar.twig
+//
+// Style guide: Components.Brand Bar
+//
+
+#su-brand-bar,
+.su-brand-bar--default {
```

Comment:
The default version does not need a modifier class. It can be simply `su-brand-bar`

Reply:
I think initially I did that and the html output by kss node was a little strange so I added the default class. Let me tweak it and try again and see what it gives me.